tap_hubspot_beta/client_base.py

```diff
@@ -177,15 +177,14 @@ def validate_response(self, response: requests.Response) -> None:
     @staticmethod
     def extract_type(field):
         field_type = field.get("type")
+        if field_type in ["string", "enumeration", "phone_number", "json", "object_coordinates"]:
```

Comment:
Wasn't in on purpose to put this after the `bool` check?

Reply:
I moved it because of [this](https://developers.hubspot.com/docs/api/crm/properties) , it says enumeration can be of type `booleancheckbox` but is a string of options separated by a comma, if goes into the bool check first we could wrongly type that as a bool.